vocabularies/nature-of-relationship.ttl

```diff
@@ -0,0 +1,242 @@
+ @prefix natofrel: <http://linked.data.gov.au/def/nature-of-relationship/> .
+ 
+ @prefix dc: <http://purl.org/dc/elements/1.1/> .
+ @prefix dct: <http://purl.org/dc/terms/> .
+ @prefix prov: <https://www.w3.org/TR/prov-o/> .
+ @prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+ @prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+ @prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+ @prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+ @prefix sdo: <http://www.schema.org/> .
+ @prefix owl: <http://www.w3.org/2002/07/owl#> .
+ @prefix prov: <http://www.w3.org/ns/prov#> .
+
+<http://linked.data.gov.au/def/nature-of-relationship> 
+a owl:Ontology , skos:ConceptScheme ;
+dct:created "2020-11-10"^^xsd:date ;
+dct:creator <http://linked.data.gov.au/org/des> ;
+dct:modified "2020-11-10"^^xsd:date ;
+dct:publisher <http://linked.data.gov.au/org/des> ;
+dct:source "Derived from Qualified Relations ISO-19115-DCTERMS and PROV-O"@en ;
+skos:definition "This is a vocabulary collection used by the DES data Catalogue to determine relationships between catalogued datasets.  It is based on the DCAT qualified relation property https://www.w3.org/TR/vocab-dcat-2/#Property:resource_qualified_relation."@en ;
+skos:prefLabel "Nature of Relationship"@en ;
```

Comment:
```suggestion
<http://linked.data.gov.au/def/dataset-relationship> 
a owl:Ontology , skos:ConceptScheme ;
dct:created "2020-11-10"^^xsd:date ;
dct:creator <http://linked.data.gov.au/org/des> ;
dct:modified "2020-11-10"^^xsd:date ;
dct:publisher <http://linked.data.gov.au/org/des> ;
dct:source "Derived from Qualified Relations ISO-19115-DCTERMS and PROV-O"@en ;
skos:definition "This is a vocabulary collection used by the DES data Catalogue to determine relationships between catalogued datasets.  It is based on the DCAT qualified relation property https://www.w3.org/TR/vocab-dcat-2/#Property:resource_qualified_relation."@en ;
skos:prefLabel "Dataset Relationship"@en ;
```
I would alter the title to be more specific about exactly what these relationships pertain to. e.g. we have [site relationships]( https://vocabs.gsq.digital/vocabulary/site-relationships) that describe the spatial and temporal relationships between types of sites, and some specialisations for particular types of sites such as boreholes.

Prefix code would also need to change along with the inScheme and topConceptOf declarations throughout.